iteration(js)/js/main.js

```diff
@@ -267,6 +267,12 @@ class User {
 }
 
 let user = null
+const local = JSON.parse(localStorage.getItem("users")).length
```

Comment:
```suggestion
const local = JSON.parse(localStorage.getItem("users")).length
```
const local = JSON.parse(localStorage.getItem("users"))?.length